Custom IParameterConsumer causes native-image to fail

Using a file with a custom parameter consumer:
```java
import picocli.CommandLine.*;
import picocli.CommandLine.Model.*;
import java.util.Stack;
import java.util.concurrent.Callable;
import picocli.CommandLine;

@Command(name = "checksum", mixinStandardHelpOptions = true, version = "checksum 4.0", description = "Prints the checksum (MD5 by default) of a file to STDOUT.")
public class Boom implements Callable<Integer> {
	@Parameters(index = "0", description = "Ignored", parameterConsumer = LocalPC.class)
	private String file;
	public static void main(String... args) {
		System.exit(new CommandLine(new Boom()).execute(args));
	}
	@Override public Integer call() throws Exception {return 0;}
}
class LocalPC implements IParameterConsumer {
	@Override public void consumeParameters(Stack<String> args, ArgSpec argSpec, CommandSpec commandSpec) {return;}
}
```


```txt
$ java -cp boom.jar:picocli-codegen-4.0.3.jar picocli.codegen.aot.graalvm.ReflectionConfigGenerator Boom  > test.json
$ native-image -jar boom.jar -H:ReflectionConfigurationFiles=test.json -H:+ReportUnsupportedElementsAtRuntime --no-server
$ ./boom --help                                                                                                                                          ] EP_BAD 4:54 PM
Exception in thread "main" picocli.CommandLine$InitializationException: Could not instantiate class LocalPC: java.lang.NoSuchMethodException: LocalPC.<init>
	at picocli.CommandLine$DefaultFactory.create(CommandLine.java:4749)
	at picocli.CommandLine$DefaultFactory.createParameterConsumer(CommandLine.java:4745)
	at picocli.CommandLine$Model$ArgSpec$Builder.<init>(CommandLine.java:7662)
	at picocli.CommandLine$Model$PositionalParamSpec$Builder.<init>(CommandLine.java:8271)
	at picocli.CommandLine$Model$PositionalParamSpec$Builder.<init>(CommandLine.java:8261)
	at picocli.CommandLine$Model$PositionalParamSpec.builder(CommandLine.java:8220)
	at picocli.CommandLine$Model$CommandReflection.buildArgForMember(CommandLine.java:9773)
	at picocli.CommandLine$Model$CommandReflection.initFromAnnotatedTypedMembers(CommandLine.java:9688)
	at picocli.CommandLine$Model$CommandReflection.initFromAnnotatedFields(CommandLine.java:9641)
	at picocli.CommandLine$Model$CommandReflection.extractCommandSpec(CommandLine.java:9546)
	at picocli.CommandLine$Model$CommandSpec.forAnnotatedObject(CommandLine.java:5118)
	at picocli.CommandLine.<init>(CommandLine.java:223)
	at picocli.CommandLine.<init>(CommandLine.java:196)
	at Boom.main(Boom.java:14)
Caused by: java.lang.NoSuchMethodException: LocalPC.<init>
	at java.lang.Class.getConstructor0(DynamicHub.java:3082)
	at java.lang.Class.getDeclaredConstructor(DynamicHub.java:2178)
	at picocli.CommandLine$DefaultFactory.create(CommandLine.java:4725)
	at picocli.CommandLine$DefaultFactory.create(CommandLine.java:4748)
	... 13 more
```

The issue seems to be that the `LocalPC` class is not put into the reflection output by the `ReflectionConfigGenerator`, and manually putting it in I get it to build and show help successfully

--- a/picocli-codegen/src/main/java/picocli/codegen/aot/graalvm/ReflectionConfigGenerator.java
+++ b/picocli-codegen/src/main/java/picocli/codegen/aot/graalvm/ReflectionConfigGenerator.java
@@ -347,6 +347,7 @@ public class ReflectionConfigGenerator {
             visitSetter(argSpec.setter());
             visitTypeInfo(argSpec.typeInfo());
             visitObjectType(argSpec.completionCandidates());
+            visitObjectType(argSpec.parameterConsumer());
             visitObjectTypes(argSpec.converters());
         }
 

